README.md

```diff
@@ -184,16 +184,21 @@ MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py install
 
 On Windows
 ```cmd
-set "VS150COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build"
+REM The folowing line is for Visual Studio 2017 Enterprise only
+set VS150COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build
+REM The folowing line is for Visual Studio 2017 Professional only
+set VS150COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build
+REM The folowing line is for Visual Studio 2017 Community only
+set VS150COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
 set CMAKE_GENERATOR=Visual Studio 15 2017 Win64
 set DISTUTILS_USE_SDK=1
 REM The following two lines are needed for Python 2.7, but the support for it is very experimental.
 set MSSdk=1
 set FORCE_PY27_BUILD=1
 REM As for CUDA 8, VS2015 Update 3 is also required to build PyTorch. Use the following line.
-set "CUDAHOSTCXX=%VS140COMNTOOLS%\..\..\VC\bin\amd64\cl.exe"
+set CUDAHOSTCXX=%VS140COMNTOOLS%..\..\VC\bin\amd64\cl.exe
```

Comment:
Why do you remove the quotes? According to this [article](https://ss64.com/nt/set.html), although it looks a bit weird, it is the right thing to do, because we should put the quotes here when there is space in the string when using set.

Reply:
Because they are erroneous. Well, ok, to be honest, they are definitely erroneous for all VS env. vars like VS150COMNTOOLS. Here you may put quotes, but only this way:
`CUDAHOSTCXX="%VS140COMNTOOLS%..\..\VC\bin\amd64\cl.exe"`

Reply:
What error does it throw? Are you using Git Bash, MinGW shell or CMD?

Reply:
Far

Reply:
Well, I tested and the problem exists. But I don't think we should remove the quotes here. They are working in CMD. It is the problem of this console emulator. I tested the command on some other emulators like `ConEmu` and it also worked. It's not acceptable that we remove the quotes here only for one console emulator, which will then cause problems for many users that use CMD.

Reply:
`set CUDAHOSTCXX="%VS140COMNTOOLS%..\..\VC\bin\amd64\cl.exe"`
And there will no problems at all.

Reply:
Result in CMD:
```cmd
Microsoft Windows [Version 10.0.17763.316]
(c) 2018 Microsoft Corporation. All rights reserved.

C:\Users\peter>set CUDAHOSTCXX="%VS140COMNTOOLS%..\..\VC\bin\amd64\cl.exe"

C:\Users\peter>echo %CUDAHOSTCXX%
"C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\..\..\VC\bin\amd64\cl.exe"

C:\Users\peter>set "CUDAHOSTCXX_OLD=%VS140COMNTOOLS%..\..\VC\bin\amd64\cl.exe"

C:\Users\peter>echo %CUDAHOSTCXX_OLD%
C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\..\..\VC\bin\amd64\cl.exe
```
Apparently, the quotes are added to the variable if I choose your option, which won't happen in the old one.

Reply:
It is ok, it works, it is a convenient common practice to enclose in quotation marks variable values, in which spaces might be presented. Compilers do it, for instance.

Reply:
It works, but not in a safe way. Let me explain it in detail. When `set "c=XXXXXXXXXXXXXXX"` is used, the quotes here are called protection quotes. It will prevent anything in the string to be interpreted as something else. We've received quite a lot of issues about this, which will complain that `\Microsoft is unexpected here`. If I choose your option, then these problems will occur again.

Reply:
Ok, let's leave it as is, moreover, `cmd` should be called beforehand.